packages/nimble-components/src/rich-text/mention-listbox/index.ts

```diff
@@ -119,6 +138,7 @@ export class RichTextMentionListbox extends FoundationListbox {
                     displayName: this.firstSelectedOption.text
                 };
                 this.emitMentionSelected(mentionDetail);
+                this.setOpen(false);
```

Comment:
Previously, `emitMentionSelected` was calling `setOpen`, which doesn't seem right. I moved the `setOpen` calls out to the call sites.

Reply:
An alternative would be to rename `emitMentionSelected` to something a more generic like `handleMentionSelected` and have it emit & call `setOpen`. Then you wouldn't have to call `setOpen(false)` right after each call to `emitMentionSelected`.

Reply:
I suppose leaving a method that does nothing but forward the argument to another method is silly. I could move `setOpen` back and move the event detail creation into it as well, but honestly, I think we can just do without this function that has two distinct responsibilities and is called in only two places.